### PR TITLE
Feature/rotary menu

### PR DESCRIPTION
**Pull request by jeanmartin**
* ScreenController now has separate queues per row
* ScreenController now supports variable columns & rows
* Implemented menu.py Class
  * Rotary encoder 1 controls which option is selected
  * Rotary encoder 2 controls the value of the current option
  * validates values with ranges
* SoundListener now takes care of keeping `noise_level_buffer` up to date
* Implemented Settings.py Class for storing settings like gpio numbers
* turned LEDControl into real Class

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+### The `experiments` folder
+
+... is for ... well, experiments!
+
+
+## Setup
+
+* [ ] `sudo apt-get install screen`
+
+* [ ] Apply screenrc
```
Comment (SimonMiaou) [inline README.md:39]: Aren't we using tmux? 👼

**Review by SimonMiaou — APPROVED** [review]
Awesome

File: README.md (hunk 1 above)
Comment (jeanmartin) [reply README.md:39]: Actually it is vim tabs inside tmux inside screen :trollface:

**Review by thebluber — APPROVED** [review]
Did you test the changes on the pi?

**Comment by jeanmartin** [comment]
no. feel free to test this branch tomorrow :) or we do it on wednesday.

**Pull request merged**